cblite/MvCommand.cc

```diff
@@ -36,10 +36,10 @@ class MvCommand : public CBLiteCommand {
 
 
     void usage() override {
-        writeUsageCommand("mv", true, "[DOC] [COLLECTION]");
+        writeUsageCommand("mv", true, "[DOC] [[SCOPE.]COLLECTION]");
```

Comment:
Since these commands are named after Unix filesystem commands, would '/' be a better delimiter than '.'?

Reply:
This command is already weirdly complicated and uses slash for a bunch of other stuff but I can give it a try.